templates/default/partials/rest.child.tmpl.partial

```diff
@@ -41,7 +41,18 @@
 {{#parameters}}
     <tr>
       <td><span class="parametername">{{#required}}*{{/required}}{{name}}</span></td>
-      <td>{{type}}</td>
+      <td>
+          {{^schema.cType}}
+            {{schema.type}}
```

Comment:
For non-complex types, I think this should rightly be `{{type}}` (and `{{format}}` below). At least that's working for me. I may make a PR for this.